NitroxPatcher/Patches/Dynamic/PrecursorDoorway_ToggleDoor_Patch.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Reflection;
+using Harmony;
+using NitroxClient.GameLogic;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.Core;
+
+namespace NitroxPatcher.Patches.Dynamic
+{
+    class PrecursorDoorway_ToggleDoor_Patch : NitroxPatch, IDynamicPatch
+    {
+        public static readonly MethodInfo TARGET_METHOD = typeof(PrecursorDoorway).GetMethod(nameof(PrecursorDoorway.ToggleDoor), BindingFlags.Public | BindingFlags.Instance);
```

Comment:
Maybe need to extract typeof(PrecursorDoorway) into a
```cs
public static readonly Type TARGET_CLASS = typeof(PrecursorDoorway);
```
to match coding style of others patches ?

Reply:
Can do though I'm sure that somebody mentioned that doing it this way was prefrence.

Reply:
This is fine. No need to have `TARGET_CLASS` if it's not gonna be used (other than redundant usage for `TARGET_METHOD`)

Reply:
We may need to do some spring clean up in all patches  then